### ISSUE TITLE
Team incomplete error from Team.get_expected_points

### ISSUE THREAD

**Issue by jack89roberts** [alan-turing-institute/AIrsenal#181]
After running `run_airsenal_optimization` with `--allow_wildcard` flag, sometimes get this error from `t.get_expected_points(next_gw, tag)`:
```
Traceback (most recent call last):
  File "/Users/jroberts/opt/anaconda3/envs/airsenalenv/bin/run_airsenal_optimization", line 11, in <module>
    load_entry_point('airsenal', 'console_scripts', 'run_airsenal_optimization')()
  File "/Users/jroberts/GitHub/AIrsenal/airsenal/scripts/fill_transfersuggestion_table.py", line 310, in main
    print_team_for_next_gw(best_strategy)
  File "/Users/jroberts/GitHub/AIrsenal/airsenal/scripts/fill_transfersuggestion_table.py", line 150, in print_team_for_next_gw
    expected_points = t.get_expected_points(next_gw, tag)
  File "/Users/jroberts/GitHub/AIrsenal/airsenal/framework/team.py", line 329, in get_expected_points
    raise RuntimeError("Team is incomplete")
RuntimeError: Team is incomplete
```
Seems to be a sporadic issue with removing/adding players (for a wildcard team), not sure of the cause. Maybe an issue with calculating sale prices or similar. Probably not wildcard specific, but more likely for a wildcard due to the number of transfers involved. Whenever I try to debug it the error seems to disappear...

Only impacts printing the starting XI (and expected points) at the end of run_airsenal_optimization, strategies compute normally.

**Comment by jack89roberts** [comment]
This is the same as #59 but for wildcard instead of free hit.

**Comment by Tdarnell** [comment]
This issue seems to also crop up after GW1 with the team that caused issue #244 - Likely cause being they have no history data for prices.

**Comment by Tdarnell** [comment]
No longer getting this after pull #248 e73347f to fix issue #244
